torch/csrc/jit/frontend/concrete_module_type.h

```diff
@@ -172,6 +173,13 @@ class VISIBILITY_HIDDEN ConcreteModuleTypeBuilder {
   // The original `nn.Module` class that we derived this ScriptModule from.
   py::object pyClass_;
 
+  // A type hint on this Module indicating the type of what it contains.
+  // This only makes sense on ModuleDict (where it might be Dict[str,
```

Comment:
I suppose since we don't allow nn.Modules as specifyable types, it always has to be a InterfaceType